### ISSUE TITLE
Autosaving notes

### ISSUE THREAD

**Issue by fabiospampinato** [notable/notable#1244]
If an editor is being used for hours continuously and the power is cut off or something there could be some data loss. Notes should probably be auto-saved.

**Comment by meichengg** [comment]
@fabiospampinato hi, is this project discontinued?

**Comment by fabiospampinato** [comment]
@meichengg No, the latest build is this one: https://github.com/notable/notable-insiders/releases/tag/v1.9.0-beta.10

I'm working on the next, and eventually on releasing stable builds back.
